### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by dnfd**

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+function getCurrencyByKey(key) {
+  if (/^eth/i.test(key))
```
Comment (shal) [inline src/helpers.js]: @denisfd This is very bad practice to make such amount of if cases, please avoid this.

Consider reading example below and fix your code 👍 

```js
function getCurrencyByKey(key) {
  const currencies = ['eth', 'btc', 'bch', 'ltc', 'xrp', 'dash', 'trst'];

  for (var currency in currencies) {
    if (/* condition */) { 
      return currency. toUpperCase();
  }
}
```

P.S: I can't find usage of this function, can you explain where it used?

File: src/fixtures/currencies.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  "Ethereums": [
+    "ETH",
+    "TRST"
+  ],
+  "BCH": {
+    "client": "bitcoincash",
+    "explorerAddress": "/address/#{addres}",
+    "explorerTransaction": "/tBCC/tx/#{txid}",
+    "baseFactor": "100000000",
+    "server": "http://admin:changeme@35.204.89.49:18332",
```
Comment (shal) [inline src/fixtures/currencies.json]: @denisfd Please delete this test. Better check that server respond `200 OK`.

File: src/fixtures/currencies.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    "gateway": "bitcoincashd",
+    "blockchainKeyPrefix": "bch"
+  },
+  "BTC": {
+    "client": "bitcoin",
+    "explorerAddress": "/address/#{address}",
+    "explorerTransaction": "/tx/#{txid}",
+    "baseFactor": "100000000",
+    "server": "http://admin:changeme@35.204.72.225:18332",
```
Comment (shal) [inline src/fixtures/currencies.json]: @denisfd Please delete this test. Better check that server respond `200 OK`.

File: src/fixtures/currencies.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    "gateway": "bitcoind",
+    "blockchainKeyPrefix": "btc"
+  },
+  "DASH": {
+    "client": "dash",
+    "explorerAddress": "",
+    "explorerTransaction": "",
+    "baseFactor": "100000000",
+    "server": "http://admin:changeme@35.204.90.220:19998",
```
Comment (shal) [inline src/fixtures/currencies.json]: @denisfd Please delete this test. Better check that server respond `200 OK`.

File: src/fixtures/currencies.json [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    "gateway": "dashd",
+    "blockchainKeyPrefix": "dash"
+  },
+  "ETH": {
+    "client": "ethereum",
+    "explorerAddress": "/address/#{address}",
+    "explorerTransaction": "/tx/#{txid}",
+    "baseFactor": "1000000000000000000",
+    "server": "http://35.198.177.238:8545",
```
Comment (shal) [inline src/fixtures/currencies.json]: @denisfd Please delete this test. Better check that server respond `200 OK`.

File: src/fixtures/currencies.json [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    "gateway": "geth",
+    "blockchainKeyPrefix": "eth"
+  },
+  "LTC": {
+    "client": "litecoin",
+    "explorerAddress": "",
+    "explorerTransaction": "",
+    "baseFactor": "100000000",
+    "server": "http://admin:changeme@35.204.246.179:19332",
```
Comment (shal) [inline src/fixtures/currencies.json]: @denisfd Please delete this test. Better check that server respond `200 OK`.

File: src/fixtures/currencies.json [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    "gateway": "litecoind",
+    "blockchainKeyPrefix": "ltc"
+  },
+  "TRST": {
+    "client": "",
+    "explorerAddress": "",
+    "explorerTransaction": "",
+    "baseFactor": "1000000",
+    "server": "http://35.198.177.238:8545",
```
Comment (shal) [inline src/fixtures/currencies.json]: @denisfd Please delete this test. Better check that server respond `200 OK`.

File: src/fixtures/currencies.json [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    "gateway": "geth",
+    "blockchainKeyPrefix": "eth"
+  },
+  "XRP": {
+    "client": "ripple",
+    "explorerAddress": "/address/#{address}",
+    "explorerTransaction": "/tx/#{txid}",
+    "baseFactor": "1000000",
+    "server": "http://admin:changeme@35.204.97.190:5005",
```
Comment (shal) [inline src/fixtures/currencies.json]: @denisfd Please delete this test. Better check that server respond `200 OK`.

File: src/helpers.js [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+  if (/^dash/i.test(key))
+    return 'DASH'
+  if (/^trst/i.test(key))
+    return 'TRST'
+
+  return 'BTC' //default
+}
+
+function getCurrencyByClient(client) {
+  switch (client) {
```
Comment (shal) [inline src/helpers.js]: Same here.

File: spec/peatio/admin/wallets.spec.js [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+            cy.get('#wallet_blockchain_key').then(field => {
+              var regex = new RegExp('^' + config.blockchainKeyPrefix)
+              assert(regex.test(field[0].value), 'Blockchain Key is valid')
+            })
+
+            cy.get('#wallet_uri').then(field => {
+              var uri = field[0].value
+              if (currencies.Ethereums.includes(code)) {
+                assert(uri == config.server || uri == 'http://geth:8545', 'URI is correct')
```
Comment (shal) [inline spec/peatio/admin/wallets.spec.js]: @denisfd I don't understand why this test should exist, server field is specific for each platform test.

File: spec/peatio/admin/wallets.spec.js [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+              }
+              else {
+                assert(uri == config.server, 'URI is valid')
```
Comment (shal) [inline spec/peatio/admin/wallets.spec.js]: @denisfd I don't understand why this test should exist, server field is specific for each platform test.

**Review by shal — CHANGES_REQUESTED** [review]

File: src/helpers.js (hunk 1 above)
Comment (dnfd) [reply src/helpers.js]: In deleted testcases :D

**Review by shal — APPROVED** [review]

**Pull request merged**